packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -1533,21 +1525,11 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
         }
     }
 
-    public async openWithEncoding(node: IZoweDatasetTreeNode): Promise<void> {
-        const encoding = await promptForEncoding(node);
-
-        const docResult = await confirmForUnsavedDoc(node);
```

Comment:
The logic on this line and below related to `confirmForUnsavedDoc` should be kept.